src/translators/crate.py

```diff
@@ -58,7 +59,7 @@ def setup(self):
         # Added backoff_factor for retry interval between attempt of
         # consecutive retries
         backoff_factor = EnvReader(log=logging.getLogger(__name__).debug) \
-            .read(FloatVar('CRATE_BACKOFF_FACTOR', 0.0))
+            .read(FloatRangeVar('CRATE_BACKOFF_FACTOR', 0.0))
```

Comment:
If we make `FloatRangeVar` generic (see comment below) then I think this line should be

```python
.read(FloatRangeVar(var_name='CRATE_BACKOFF_FACTOR', default_value=0.0, lo=0.0, hi=120.0)
```